src/main/webapp/components/workspaces/subscribe.js

```diff
@@ -4,6 +4,7 @@ import IconButton from '@material-ui/core/IconButton'
 import { CustomTooltip } from '../tooltip/tooltip'
 import EmailOutlinedIcon from '@material-ui/icons/EmailOutlined'
 import { getIn } from 'immutable'
+import { MetacardInteraction } from '../metacard-interaction'
 
 const subscribeToWorkspace = async props => {
   const { id, title, setMessage, subscribe, setSubscribed } = props
```

Comment:
I think for CI to pass you just need to delete this line so that these aren't unused variables anymore